docs/src/components/Tabs/index.tsx

```diff
@@ -27,8 +23,9 @@ const StyledContainer = styled.div<{ collapsed: boolean }>`
     z-index: 1; /* place tabs above the main container */
     display: flex;
     flex-wrap: wrap; /* so that we can detect whether tabs wrap, and in that case collapse them */
-    padding-top: ${SHADOW_PADDING_TOP};
-    padding-right: ${SHADOW_PADDING_RIGHT};
+    padding-top: ${getTabShadowReachTop(theme)};
```

Comment:
you don't need to pass the theme as a parameter explicitly, change the function, so you can just pass the function as a reference, as we usually do with SC.